# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/meson/no-ccache.patch
# Copyright (C) 2025 - 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- meson-1.10.0/mesonbuild/envconfig.py.vanilla	2025-12-09 20:28:46.796000000 +0100
+++ meson-1.10.0/mesonbuild/envconfig.py	2025-12-09 20:30:22.750416591 +0100
@@ -436,11 +436,11 @@
 
     @staticmethod
     def detect_ccache() -> ExternalProgram:
-        return ExternalProgram('ccache', silent=True)
+        return ExternalProgram('ccache_doesnt_exist', silent=True)
 
     @staticmethod
     def detect_sccache() -> ExternalProgram:
-        return ExternalProgram('sccache', silent=True)
+        return ExternalProgram('sccache_doesnt_exist', silent=True)
 
     @staticmethod
     def detect_compiler_cache() -> ExternalProgram:
